src/util/commonServiceRequest.ts

```diff
@@ -84,3 +88,52 @@ export const simpleCall = async (url: string, method: HTTPMethod = "GET", data:
 	}
 	throw new Error(content.message);
 };
+
+export async function multipartFormDataRequest <A>(
+	method: HTTPMethod = "POST",
+	url: string,
+	dataDecoder: t.Type<A>,
+	parameters: JSONObject
+): Promise<Either<CommonServiceRequestError, A>>{
+
+	let response: Response;
+	const formData = new FormData();
+	
+	Object.keys(parameters).forEach((key) => (formData.append(key, parameters[key])));
```

Comment:
Cuando no vas a hacer algo con cada elemento del arreglo en cuestion, es mejor evitar el forEach que esta mas dedicado a eso, en su lugar podes:
```suggestion
for (const key in parameters) {
  formData.append(key, parameters[key])
}
```